mitosheet/mitosheet/code_chunks/step_performers/import_steps/excel_range_import_code_chunk.py

```diff
@@ -10,52 +10,87 @@
 from mitosheet.code_chunks.code_chunk import CodeChunk
 from mitosheet.excel_utils import (get_column_from_column_index,
                                    get_col_and_row_indexes_from_range)
+from mitosheet.public.v2.excel_utils import get_read_excel_params_from_range
 from mitosheet.state import State
+from mitosheet.transpiler.transpile_utils import column_header_to_transpiled_code
+from mitosheet.types import ExcelRangeImport
+
+
+EXCEL_RANGE_IMPORT_TYPE_RANGE = 'range'
+EXCEL_RANGE_IMPORT_TYPE_UPPER_LEFT = 'upper left corner value'
```

Comment:
EXCEL_RANGE_IMPORT_TYPE_UPPER_LEFT -> EXCEL_RANGE_IMPORT_TYPE_UPPER_LEFT_VALUE for consistency